assignment2/company.rb

```diff
@@ -0,0 +1,95 @@
+require 'csv'
+
+module Designation
+    def self.included(klass)
+        klass.extend(ClassMethods)
+    end
+
+    module ClassMethods
+        def find_experience(experience)
+            if (experience > 5)
+                puts "Enployee Designation: Team Lead"
+            else
+                puts "Employee Designation: Developer"
+            end
+        end
+    end
+
+    def find_experience_obj(experience)
+        if (experience > 5)
+            puts "Enployee Designation: Team Lead"
+        else
+            puts "Employee Designation: Developer"
+        end
+    end
+end
+
+class Company
+    include Designation
+    def initialize(id, name, addr, experience, salary, stack)
+        @emp_id = id
```

Comment:
Can you create an `Employee` class and initialize employee objects using the `Employee` class. You can also add inheritance to it(ex: `Developer` class, `Manager` class and `HR` class).